modules/apps/transfer/keeper/grpc_query.go

```diff
@@ -121,3 +121,22 @@ func (q Keeper) EscrowAddress(c context.Context, req *types.QueryEscrowAddressRe
 		EscrowAddress: addr.String(),
 	}, nil
 }
+
+// TotalEscrowForDenom implements the TotalEscrowForDenom gRPC method.
+func (q Keeper) TotalEscrowForDenom(c context.Context, req *types.QueryTotalEscrowFormDenomRequest) (*types.QueryTotalEscrowForDenomResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "empty request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+
+	if q.IsIBCDenom(ctx, req.Denom) {
+		return nil, status.Error(codes.InvalidArgument, "denom is not a native token denomination")
+	}
```

Comment:
I think this can be removed now? The associated test case should be expected to pass